src/Api/Helpers/AddDatabaseExtensionMethod.cs

```diff
@@ -14,27 +14,23 @@ public static class AddDatabaseExtensionMethod
     public static IServiceCollection AddDatabase(this IServiceCollection services, IConfiguration configuration)
     {
         // Database information
-        var sqlite = configuration.GetConnectionString("sqlite");
-        var mssql = configuration.GetConnectionString("mssql");
-
-        sqlite = configuration.GetValue<string>("ConnectionStrings:Sqlite:Api", sqlite);
-        mssql = configuration.GetValue<string>("ConnectionStrings:Mssql:Api", mssql);
+        var sqlite = configuration.GetConnectionString("sqlite:api");
+        var mssql = configuration.GetConnectionString("mssql:api");
 
         if (!string.IsNullOrEmpty(sqlite))
         {
             services.AddDbContext<DbTenantContext, SqliteContext>((sp, builder) =>
             {
-                var config = sp.GetRequiredService<IConfiguration>();
-                builder.UseSqlite(config.GetConnectionString("sqlite"));

```

Comment:
It's fine to remove this but I'll explain why I did this and leave the decision up to you. By using `IConfiguration` from the service provider instead of the outer scope one you are able to keep this method `static`. This way it won't capture any outer scope, requires less allocations, and should run a tad faster.

There are no bugs that should arise from capturing `IConfiguration` because it's a singleton, it's lifetime will match that of the Host so it won't unnecessarily be kept from being GC'ed (because it won't ever be), and changes to it aren't done through creating a new reference. So in this instance it's safe but I generally try to not capture scope if I can help it. 

Decision is up to you though.

Reply:
Aha I see, yeah that's a valid point. Will add it back with a explainer.